ruby/vampires.rb

```diff
@@ -56,5 +56,4 @@
   end

```

Comment:
Inside your line 47 loop:

This section should happen before the final vampire evaluations.  Try to keep your logic organized in a way that the program collects all of the necessary data first and then does a full evaluation.

Collecting the allergy data first and then running returning the vampire results all at once is a better practice vs collecting some of the data, returning an eval and then re-evaluating.